biothings/hub/databuild/backend.py

```diff
@@ -154,14 +154,24 @@ def get_src_metadata(self):
             # are sub-source names, not main source so we have to deal with src_dump as well
             # in order to resolve/map main/sub source name
             subsrc_versions = []
+
+            if src and src.get("download"):
+                # Store the latest dump time
+                src_meta.setdefault(src["_id"], {}).setdefault("dump_date", src["download"]["started_at"])
```

Comment:
@sengineer0 let's use `download_date` instead of `dump_date` here.

Reply:
@sengineer0 there is another issue here. What if `src["download"]["status"]` is not `success`? 

I think we should only take the timestamp from the success dump.

Now the question is if `failed`, do we have the timestamp from the last success dump?